module/VuFind/src/VuFind/ILS/Driver/Symphony.php

```diff
@@ -1152,15 +1147,23 @@ public function patronLogin($username, $password)
                         $addrinfo->addressPolicyID == 'EMAIL'
                         && !empty($addrinfo->addressValue)
                     ) {
-                        $patron['email'] = $addrinfo->addressValue;
+                        $email = $addrinfo->addressValue;
                         break;
                     }
                 }
             }
         }
 
         // @TODO: major, college
-
+        $patron = $this->createPatronArray(
+            id: $resp->patronInfo->$usernameField,
+            firstname: $firstName,
+            lastname: $lastName,
+            email: $email,
+            cat_username: $username,
+            cat_password: $password
+        );
+        $patron['library'] = $resp->patronInfo->patronLibraryID;
```

Comment:
Maybe worth adding a comment here:

```suggestion
        // TODO: if we expose this as home_library through getMyProfile, it may be possible to improve
        // patron home library selection functionality; would require testing.
        $patron['library'] = $resp->patronInfo->patronLibraryID;
```